### DISCUSSION TITLE
Populating LLDP neighbors , status, configuration tabs without NAPALM

### DISCUSSION THREAD

**Discussion by saarbali01** [netbox-community/netbox#11451] [Q&A] [answered]
I cant enable NXAPI in my environment , is it possible to populate LLDP neighbors , status, configuration tabs via another way such like a python script or anything else but not NAPALM?

**Comment by mtinberg** [comment, marked as answer]
I'm pretty sure NAPALM can talk to nxos_ssh instead of NXAPI


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: Shadi Arbali ***@***.***>
Sent: Tuesday, January 10, 2023 7:38 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Populating LLDP neighbors , status, configuration tabs without NAPALM (Discussion #11451)


I cant enable NXAPI in my environment , is it possible to populate LLDP neighbors , status, configuration tabs via another way such like a python script or anything else but not NAPALM?

**Reply by saarbali01** [reply]
Didnt realize we have that option , i changed the platform argument to nxos-ssh and it worked , Thank you!
